src/Patterns/Patterns.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Patterns;
+
+use Automattic\WooCommerce\Blocks\Domain\Package;
+
+/**
+ * Patterns class
```

Comment:
Let's add `@internal` here to indicate its for internal use only. This is something we did in the [BlockTemplatesController](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/BlockTemplatesController.php#L10) since other Woo plugins tried piggybacking on the functionality.